lib/cfn-nag/custom_rules/IamPolicyPassRoleWildcardResourceRule.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class IamPolicyPassRoleWildcardResourceRule < BaseRule
```

Comment:
consider deriving this from a base class to share the code with managedpolicy

Reply:
Base class created